README.md

```diff
@@ -23,6 +28,13 @@ The Azure App Services extension for VS Code lets you quickly browse, create, ma
 
 * View and edit a web app's files
   * To enable this feature, modify your `appService.showRemoteFiles` user setting to true.
+  * To modify user settings, click File > Preferences > Settings.
```

Comment:
Can you combine this bullet with the one above? Something like:
To enable this feature, click File > Preferences > Settings and modify `appService.showRemoteFiles` to be true.